Dockerfile

```diff
@@ -1,3 +1,21 @@
-FROM nginxinc/nginx-unprivileged:mainline
-USER nginx
-COPY ./dist /usr/share/nginx/html
\ No newline at end of file
+FROM node:24-alpine AS builder
+
+WORKDIR /app
+
+RUN corepack enable
+
+COPY pnpm-lock.yaml package.json .
+RUN pnpm fetch
+
+COPY src src
+COPY public public
+COPY *.json *.ts *.cjs *.mts *.html .
+RUN pnpm install
+RUN pnpm build
+
+FROM caddy:2-alpine
+
+COPY Caddyfile /etc/caddy/Caddyfile
+COPY --from=builder /app/dist /srv
+
+# NOTE: Caddy will serve on port 80
```

Comment:
For deployments, post 80 is taken either by traefik or ingess-nginx balancer. Is it possible to serve web server on 8080 port?

Reply:
Sure, changed to 8080.